runtime/moonriver/src/lib.rs

```diff
@@ -156,11 +159,14 @@ parameter_types! {
 	/// We allow for one half second of compute with a 6 second average block time.
 	/// These values are dictated by Polkadot for the parachain.
```

Comment:
This comment seems kind of out of place now.

Reply:
honestly, this topic could easily have it's own `.md` file